### ISSUE TITLE
Tracking issue for v1

### ISSUE THREAD

**Issue by Thomasdezeeuw** [tokio-rs/mio#1533]
I've set a tentative deadline for Mio v1 on February 1st 2022. Any issue fixed before that are tracking in [v1.0 milestone](https://github.com/tokio-rs/mio/milestone/4).

That gives us roughly 2 and half months to fix any bugs that appear in v0.8, although I don't expect many will since the implementation hasn't changed much since v0.7.

I don't expect any more changes to the API with only one possible addition: adding API/hooks for IOCP https://github.com/tokio-rs/mio/pull/1345. But I expect that will strictly be an addition and doesn't need to change existing API.

**Comment by Noah-Kennedy** [comment]
@Thomasdezeeuw Do we want to wait on v1 for figuring out io_uring support, if this is something we are interested in?

**Comment by Thomasdezeeuw** [comment]
> @Thomasdezeeuw Do we want to wait on v1 for figuring out io_uring support, if this is something we are interested in?

I took too long with v1, so I started in https://github.com/Thomasdezeeuw/a10 with io_uring.

**Comment by Thomasdezeeuw** [comment]
Closing this in favour of the milestone: https://github.com/tokio-rs/mio/milestone/4.
